src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form 
+    data-qa="big" 
+    class="search-form" 
+    action="/" 
+    method="get"
+    >
+      
+      <label> 
+        <input 
+        type="search"
+        id="text1" 
+        class="input big-input" 
+        data-qa="keypress" 
+        placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
```

Comment:
no need for id, if you wrap input in label

```suggestion
      <label> 
         <input 
            type="search"
            class="input big-input" 
            data-qa="keypress" 
            placeholder="Try &quot;Los Angeles&quot;"
          >
      </label>
```